src/ports/squids/component.ts

```diff
@@ -168,7 +168,39 @@ export async function createSubsquidComponent({
 
     // NOTE: in the future, depending on the project we might want to run the promote query in a different db
     await dappsDatabase.query(promoteQuery)
-    console.log(`The ${serviceName} was promoted and the active schema is ${schemaName}`) // @TODO implement a proper response
+    console.log(`The ${serviceName} was promoted and the active schema is ${schemaName}`)
+
+    // Call marketplace server to recreate triggers and refresh materialized view
+    try {
+      const marketplaceServerUrl = await config.getString('MARKETPLACE_SERVER_URL')
```

Comment:
Will this new code be running the promotion on any deployment, event the ones that don't belong to the marketplace? Is there a way to change this based on the schema name?